consensus/hotstuff/committees/consensus_committee.go

```diff
@@ -65,19 +66,20 @@ func NewConsensusCommittee(state protocol.State, me flow.Identifier) (*Consensus
 }
 
 func (c *Consensus) Identities(blockID flow.Identifier, selector flow.IdentityFilter) (flow.IdentityList, error) {
-	return c.state.AtBlockID(blockID).Identities(filter.And(
+	il, err := c.state.AtBlockID(blockID).Identities(filter.And(
 		filter.IsVotingConsensusCommitteeMember,
 		selector,
 	))
+	return il, convertError(err)
 }
 
 func (c *Consensus) Identity(blockID flow.Identifier, nodeID flow.Identifier) (*flow.Identity, error) {
 	identity, err := c.state.AtBlockID(blockID).Identity(nodeID)
 	if err != nil {
-		return nil, fmt.Errorf("could not get identity for node ID %x: %w", nodeID, err)
+		return nil, fmt.Errorf("could not get identity for node ID %x: %w", nodeID, convertError(err))
```

Comment:
same reasoning: I feel we could just do the error conversion explicitly here and remove `convertError`